content/topics/work/mock-interview/_index.md

```diff
@@ -0,0 +1,13 @@
+---
+ready: true
+title: Mock Interviews
+---
+### Mock Interviews
```

Comment:
Repetitive headings. You did not look at this page.

`title: Mock Interviews` This gets rendered as an h1 heading
`### Mock Interviews` This gets rendered as an h3 heading.

So your final rendered content looks like:

# Mock Interviews
### Mock Interviews


This is bad!

Make it look like this instead

# Mock Interviews